com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/shutdown/model/ShutdownDto.java

```diff
@@ -0,0 +1,81 @@
+package com.sap.cloud.lm.sl.cf.core.shutdown.model;
+
+public class ShutdownDto {
+    private Status status = Status.RUNNING;
+    private String appId;
+    private String appInstanceId;
+    private String appInstanceIndex;
+    private long cooldownTimeoutInSeconds;
+
+    private ShutdownDto(Status status, String appId,
```

Comment:
AsyncExecutorShutdownDto

Reply:
Actually the purpose of this DTO is to represent the shutdown state of the whole app, not just the Flowable JobExecutor, thats why it is named ShutdownDto.